<launch>
    <arg name="gnss_port_name" default="/dev/ttyTHS0" />
    <arg name="gnss_baud_rate" default="230400" />
    <arg name="gnss_nmea_head" default="GPCHC" />
    <arg name="gnss_org_lat" default="32.02778534" />
    <arg name="gnss_org_lon" default="118.8564381" />
    <arg name="gnss_is_pub_sentence" default="true" />
    <arg name="gnss_is_pub_pose" default="true" />
    <arg name="gnss_is_broadcast_tf" default="false" />
    <arg name="gnss_sentence_topic" default="nmea_sentence" />
    <arg name="gnss_pose_topic" default="gnss_pose" />
    <node name="hc_nmea_driver" pkg="nmea_driver" type="nmea_driver" respawn="true" output="screen">
        <param name="port_name" type="string" value="$(arg gnss_port_name)" />
        <param name="baud_rate" type="int" value="$(arg gnss_baud_rate)" />
        <param name="nmea_head" type="string" value="$(arg gnss_nmea_head)" />
        <param name="org_lat" type="double" value="$(arg gnss_org_lat)" />
        <param name="org_lon" type="double" value="$(arg gnss_org_lon)" />
        <param name="is_pub_sentence" type="bool" value="$(arg gnss_is_pub_sentence)" />
        <param name="is_pub_pose" type="bool" value="$(arg gnss_is_pub_pose)" />
        <param name="is_broadcast_tf" type="bool" value="$(arg gnss_is_broadcast_tf)" />
        <param name="sentence_topic" type="string" value="$(arg gnss_sentence_topic)" />
        <param name="pose_topic" type="string" value="$(arg gnss_pose_topic)" />
    </node>
</launch>
